examples/nemo/eg1/README.md

```diff
@@ -2,18 +2,20 @@
 
 **Authors:** A. R. Porter, STFC Daresbury Lab
 **Modified by:** R. W. Ford, STFC Daresbury Lab
+**Modified by:** S. Siso, STFC Daresbury Lab
 
-This directory contains an example script, `runme_openmp.py`, that
-demonstrates the use of PSyclone to parallelise all loops over levels
-for the `tra_adv.F90` code, using OpenMP. Once you have installed
-PSyclone, this script may be run by doing:
+This directory contains the example scripts: `runme_openmp.py` and
+`runme_openmp_gpu.py`, that demonstrates the use of PSyclone to parallelise
+all loops over levels for the `tra_adv.F90` code, using OpenMP. Once you have
+installed PSyclone, this scripts may be run by doing:
 
 ```sh
-./runme_openmp.py
+./runme_openmp{_gpu}.py
```

Comment:
Sorry to be a pain but we're trying to move away from this form of example in favour of providing a PSyclone transformation script (so that this line would become, e.g. `psyclone -api nemo -s ./openmp_gpu_trans.py tra_adv.F90`. Please could you do that for the new example you've added and, if you're feeling generous, the existing one too.

Reply:
Done for both scripts.

Reply:
Gold star :-) Thanks very much.